charts/opl-content-api/dev-values.yaml

```diff
@@ -27,6 +27,10 @@ deployment:
       value: "production"
     - name: "DATABASE_HOST"
       value: "opl-mongodb"
+    - name: "OPL_CONTENT_API_BASE_URL"
+      value: "http://opl-content-api-opl-dev.apps.s44.core.rht-labs.com"
+    - name: "AMA_DEST_EMAIL"
+      value: "openpracticelibraryteam@gmail.com"
```

Comment:
We don't actually use the helm charts anymore. I left them here for reference to the original deployment, and we do use the `mongodb-values.yaml` to deploy mongodb in Openshift.

Reply:
OK - Removing new values from helm chart.